overlord/devicestate/firstboot.go

```diff
@@ -97,11 +97,11 @@ func populateStateFromSeedImpl(st *state.State, opts *populateStateFromSeedOptio
 	mode := "run"
 	sysLabel := ""
 	preseed := false
-	hasModeenv := false
+	isCoreBoot := false
 	if opts != nil {
 		if opts.Mode != "" {
 			mode = opts.Mode
-			hasModeenv = true
+			isCoreBoot = true
```

Comment:
maybe it was confusing, but my ask was not to just change the name, this is not the definition of core boot, we also need to set if !release.OnClassic

Reply:
Oh, sorry, I just renamed without stopping one sec for thinking. Changed.